qa/L0_storage_S3/test.sh

```diff
@@ -390,6 +390,27 @@ wait $SERVER_PID
 aws s3 rm "${BUCKET_URL_SLASH}" --recursive --include "*"
 aws s3 rb "${BUCKET_URL}"
 
+# Test access decline error message
+export AWS_SECRET_ACCESS_KEY="[Invalid]" && export AWS_SESSION_TOKEN=""
+SERVER_ARGS="--model-repository=$BUCKET_URL --exit-timeout-secs=120"
+run_server
+if [ "$SERVER_PID" != "0" ]; then
+    echo -e "\n***\n*** Unexpected server start $SERVER\n***"
+    cat $SERVER_LOG
+    kill $SERVER_PID
+    wait $SERVER_PID
+    RET=1
+else
+  # AWS S3 does not appear to reply on access decline, but other implementations
+  # might provide extra messages, so make sure Triton will print the messages.
```

Comment:
Can we add similar tests to https://github.com/triton-inference-server/server/blob/main/qa/L0_storage_S3_local/test.sh to see how an on-prem S3 implementation replies? I wonder if it is different.

Reply:
Yes. [Add MinIO S3 access decline test](https://github.com/triton-inference-server/server/pull/5825/commits/0a8310369d4df97c1acdb279b26ce24de68c5a50)

Reply:
It looks like the response is the same, "No response body."